pages/Ethereum/Identity.md

```diff
@@ -1,48 +1,28 @@
 ---
```

Comment:
Updated this page to be specific for Ethereum. The chain agnostic content moved to pages/DSNP/Identity.md